styles/theme.ts

```diff
@@ -3,17 +3,17 @@ import { breakpointTablet, breakpointDesktop } from "./vars"
 export const colors = {
   error: "#DC0C31",
   link: "#0029FF",
+  white: "#FFFFFF",
   //   neutral colors
   mainBtnText: "#FCFCFC",
   disabledBtnBg: "#CECECE",
-  darkGray: "#BFBFBF",
-  charcoal: "#B1B1B1",
   mainPlaceholder: "#939393",
-  steelGray: "#585858",
   disabledBtnText: "#464646",
   mainLabel: "#353535",
   mainText: "#232323",
   mainTitle: "#121212",
+  //   primary color
+  mainAccent: "#FFBD00",
```

Comment:
maybe we named vars as in maket and then it will be easier to substitute variables

Reply:
for example?